specificatie/verblijfplaats.yaml

```diff
@@ -452,51 +361,27 @@ components:
               type: boolean
             woonplaats:
               type: boolean
-            adresregel1:
-              type: boolean
-            adresregel2:
-              type: boolean
-            indicatieVestigingVanuitBuitenland:
-              type: boolean
-            vanuitVerblijfplaatsOnbekend:
-              type: boolean
     VerblijfplaatsOnbekendInOnderzoek:
       allOf:
         - $ref: 'persoon.yaml#/components/schemas/InOnderzoek'
         - type: object
           properties:
-            datumInschrijvingInGemeente:
-              type: boolean
-            gemeenteVanInschrijving:
+            functieAdres:
               type: boolean
             verblijfplaatsOnbekend:
```

Comment:
verblijfplaatsOnbekend is verwijderd uit VerblijfplaatsOnbekend, dus kan ook verwijderd worden uit VerblijfplaatsOnbekendInOnderzoek

Reply:
Aangepast